sync/worker.go

```diff
@@ -158,6 +158,7 @@ func NewFetchWorker(s WorkerInfra, count int, reqFactory BatchRequestFactory, id
 				idsStr := concatShortIds(remainingItems)
 				lg.Info("send fetch request to Peer: %v ids: %v", peer.String(), idsStr)
 				ch, _ := reqFactory(s, peer, remainingItems)
+				lg.Info(concatShortIds(remainingItems))
```

Comment:
Do you calculate it twice? concatShortIds is called twice

Reply:
redundant print